lib/sorteios_web/live/components.ex

```diff
@@ -97,6 +97,33 @@ defmodule SorteiosWeb.Components do
         </div>
       </div>
     </div>
+
+    <footer>
```

Comment:
Can we create a component called `common_footer` and use it on both layouts so we make it DRY?

Reply:
Sure! Good idea 👍

Reply:
I saw that you used it as a function rather than a component. Can you pass the class as an assign?

<.common_footer class="..." />